src/commands/aqua.ts

```diff
@@ -190,9 +209,48 @@ export default class Aqua extends Command {
             watchingNotification();
           })
           .catch((error): void => {
-            this.log(error);
+            commandObj.log(String(error));
             return watchingNotification();
           });
       });
   }
 }
+
+const resolveOutputPath = (
+  projectRootDir: string,
+  maybeOutputPath: string | undefined
+): string | undefined => {
+  if (maybeOutputPath === undefined) {
+    return undefined;
+  }
+
+  const outputPath = maybeOutputPath;
+
+  if (path.isAbsolute(outputPath)) {
+    return outputPath;
+  }
+
+  return path.resolve(projectRootDir, outputPath);
+};
+
+const addFileExtensionsInTsFiles = async (outputDirPath: string) => {
```

Comment:
This must be updated in `Aqua` directly instead of trying to overwrite the content of generated files.
Moreover are you sure it's time to migrate JS\TS projects (not the CLI itself, but the generated code) to ESM. We haven't released the new JS Client yet

Reply:
This is a temporary measure so it works at least. I agree with you. Also I think maybe compilation to js should be done by fluence cli and aqua should only be concerned with producing air